### PR TITLE
disassemble module 69

### PR DESCRIPTION
**Pull request by red031000**
nice

### CODE REVIEW

File: arm9/modules/69/asm/module_69.s [hunk 1]
```diff
@@ -0,0 +1,4141 @@
...
+	add r0, r4, #0
+	bl FUN_02066A20
+	add r0, r5, #0
+	pop {r3, r4, r5, r6, r7, pc}
+	nop
+_0222F788: .word 0x00001B54
+_0222F78C: .word 0x00001B58
+
+    .section .rodata
+    .incbin "baserom.nds", 0x2c49d0, 0x1ac
```
Comment (PikalaxALT) [inline arm9/modules/69/asm/module_69.s:4135]: Address comment (0x0222F790)

File: arm9/modules/69/asm/module_69.s [hunk 2]
```diff
@@ -0,0 +1,4141 @@
...
+
+    .section .data
```
Comment (PikalaxALT) [inline arm9/modules/69/asm/module_69.s:4137]: Address comment (0x0222F940)

File: arm9/modules/69/asm/module_69.s [hunk 3]
```diff
@@ -0,0 +1,4141 @@
...
+    .word 0x00000001
+
+    .section .bss
+    .space 0x20
```
Comment (PikalaxALT) [inline arm9/modules/69/asm/module_69.s:4141]: Missing newline at end of file

**Comment by martmists** [comment]
Obligatory "Nice"

**Pull request merged**